lib/notification-issue.coffee

```diff
@@ -183,6 +183,12 @@ class NotificationIssue
         packagePaths[packageName] = fs.realpathSync(packagePath)
 
     getPackageName = (filePath) =>
+      filePath = /(\((.+?):\d+|\((.+)\)|(.+))/.exec(filePath)[1]
+
+      # Stack traces may be a file URI
+      if /file:\/\/(\w*)\/(.*)/.test(filePath)
+        filePath = /file:\/\/(\w*)\/(.*)/.exec(filePath)[2]
+
```

Comment:
It looks like the same regex is created and executed twice here, and the first capture group is unused. Also, sometimes when regexps have multiple `/` characters, it's easier to write them using the `RegExp` constructor than as literals. I think it'd be nice to do it like this:

```coffee
# somewhere at the top of the file
FileURLRegExp = new RegExp('file://\w*/(.*)')

# ...

   if match = FileURLRegExp.exec(filePath)
     filePath = match[1]
```